onnx/version_converter/adapters/clip_10_11.h

```diff
@@ -0,0 +1,64 @@
+// Adapter for Clip in default domain from version 10 to 11
+
+#pragma once
+#include <limits>
+
+namespace ONNX_NAMESPACE { namespace version_conversion {
+
+class Clip_10_11 final : public Adapter {
+  public:
+    explicit Clip_10_11()
+      : Adapter("Clip", OpSetID(10), OpSetID(11)) {}
+
+    void adapt_clip_10_11(std::shared_ptr<Graph> graph, Node* node) const {
+      bool has_min = node->hasAttribute(kmin);
+      bool has_max = node->hasAttribute(kmax);
+
+      // Turn min/max attributes into tensor (if present) and add value as input
+      if(has_min) {
+        addMinInit(graph, node);
+      }
+      if(has_max) {
+        if(!has_min) {addMinDefault(graph, node);}
+        addMaxInit(graph, node);
+      }
+    }
+
+    void addMinInit(std::shared_ptr<Graph> graph, Node *node) const {
```

Comment:
Do you need 3 methods? You can simply create 1 generic method which takes a few extra params for the values of min, max, default no?

Reply:
Of course! I made this when I was starting getting familiar with the system and wanted to keep things very simple. Changed it to use the generic method.